Plugins/BezierSolution/BezierSpline.cs

```diff
@@ -620,9 +620,9 @@ public Vector3 GetPoint( float normalizedT )
 			{
 				// 2nd conditions isn't 'else if' because in rare occasions, floating point precision issues may arise; e.g. for normalizedT = -0.0000000149,
 				// incrementing the value by 1 results in perfect 1.0000000000 with no mantissa
-				if( normalizedT < 0f )
+				while ( normalizedT < 0f )
```

Comment:
Since the changes are small, please follow the same coding style: `while( normalizedT < 0f )`